catchup/ledgerFetcher.go

```diff
@@ -74,13 +74,18 @@ func makeLedgerFetcher(net network.GossipNode, accessor ledger.CatchpointCatchup
 }
 
 func (lf *ledgerFetcher) requestLedger(ctx context.Context, peer network.HTTPPeer, round basics.Round, method string) (*http.Response, error) {
-	parsedURL, err := network.ParseHostOrURL(peer.GetAddress())
-	if err != nil {
-		return nil, err
-	}
+	var ledgerURL string
+	if network.IsMultiaddr(peer.GetAddress()) {
```

Comment:
Isn't in master, so I think this function was added in an earlier PR to the branch.

Reply:
correct, was added earlier